contracts/AllDay.cdc

```diff
@@ -31,6 +33,16 @@ import NonFungibleToken from "./NonFungibleToken.cdc"
 // The AllDay NFTs and metadata contract
 //
 pub contract AllDay: NonFungibleToken {
+
+    //------------------------------------------------------------
+    // Deployment variables
+    //------------------------------------------------------------
+
+    // The address to which royalties should be deposited
+    pub fun RoyaltyAddress() : Address {
```

Comment:
Do we need to have a RoyaltyAddress in order to implement the metadata views?

Reply:
No, I'll get rid of this method and move the variable into `resolveView()`